iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/AddInstruction.java

```diff
@@ -19,5 +19,13 @@
  */
 package org.sonar.iac.docker.tree.api;
 
+/**
+ * AddInstruction
+ * This interface define the contract of the <a href="https://docs.docker.com/engine/reference/builder/#add">ADD</> instruction.
```

Comment:
```suggestion
 * Interface to define the contract of the <a href="https://docs.docker.com/engine/reference/builder/#add">ADD</> instruction.
```
IMO you don't need `this` all the time. Just use the neutral form.